orc8r/cloud/go/services/metricsd/metricsd/main.go

```diff
@@ -48,7 +48,8 @@ func main() {
 	}
 
 	controllerServicer := servicers.NewMetricsControllerServer()
-	protos.RegisterMetricsControllerServer(srv.GrpcServer, controllerServicer)
+	protos.RegisterMetricsControllerSouthboundServer(srv.GrpcServer, controllerServicer.Southbound)
+	protos.RegisterMetricsControllerProtectedServer(srv.GrpcServer, controllerServicer.Protected)
```

Comment:
Yeah for reference -- this is the key point here, that we're registering these servicers separately. That way, we'll be able to register them to different GrpcServers when we add that functionality